### PR TITLE
fix(account): update how to change the account language

### PR DESCRIPTION
**Pull request by leannatherese**
### Description

Removed steps that were repeated twice.

### CODE REVIEW

File: console/account/how-to/change-language.mdx [hunk 1]
```diff
@@ -20,12 +20,9 @@ You can choose which language you would like to view the [Scaleway console](/con
...
 
 1. Log into the [Scaleway console](https://console.scaleway.com).
-    <Lightbox src="scaleway-console-menu.webp" alt="" />
-2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A).1. Log into the [Scaleway console](https://console.scaleway.com).
-    <Lightbox src="scaleway-console-menu.webp" alt="" />
-2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A).
-4. Click the **Edit Profile** button. The **Edit Profile** screen displays:
-    <Lightbox src="scaleway-edit-profile.webp" alt="" />
-5. Choose your preferred language from the drop-down list, and click **Confirm Changes**.
+2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays.
```
Comment (leannatherese) [inline console/account/how-to/change-language.mdx:23]:
```suggestion
2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays.
<Lightbox src="scaleway-console-menu.webp" alt="" />
```

Comment (leannatherese) [reply console/account/how-to/change-language.mdx:23]: fix(account): add image

File: console/account/how-to/change-language.mdx [hunk 2]
```diff
@@ -20,12 +20,10 @@ You can choose which language you would like to view the [Scaleway console](/con
...
```
Comment (leannatherese) [inline console/account/how-to/change-language.mdx:23]:
```suggestion
2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays:
```

File: console/account/how-to/change-language.mdx [hunk 3]
```diff
@@ -20,12 +20,10 @@ You can choose which language you would like to view the [Scaleway console](/con
...
 1. Log into the [Scaleway console](https://console.scaleway.com).
-    <Lightbox src="scaleway-console-menu.webp" alt="" />
-2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A).1. Log into the [Scaleway console](https://console.scaleway.com).
-    <Lightbox src="scaleway-console-menu.webp" alt="" />
-2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A).
-4. Click the **Edit Profile** button. The **Edit Profile** screen displays:
-    <Lightbox src="scaleway-edit-profile.webp" alt="" />
-5. Choose your preferred language from the drop-down list, and click **Confirm Changes**.
+2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays:
+<Lightbox src="scaleway-console-menu.webp" alt="" />
```
Comment (leannatherese) [inline console/account/how-to/change-language.mdx:24]:
```suggestion
 <Lightbox src="scaleway-console-menu.webp" alt="" />
```

**Review by nerda-codes — APPROVED** [review]

File: console/account/how-to/change-language.mdx [hunk 4]
```diff
@@ -20,12 +20,10 @@ You can choose which language you would like to view the [Scaleway console](/con
...
-    <Lightbox src="scaleway-edit-profile.webp" alt="" />
-5. Choose your preferred language from the drop-down list, and click **Confirm Changes**.
+2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays:
+ <Lightbox src="scaleway-console-menu.webp" alt="" />
+3. In the User information section, click the **Edit profile** button. The **Edit your account profile** screen displays:
+ <Lightbox src="scaleway-edit-profile.webp" alt="" /> 
+4. Choose your preferred language from the drop-down list, then click **Confirm changes**.
 
-    Your console will now be displayed in your preferred language, and you will also receive marketing and product communications in this language. You can repeat this procedure to change your preferred language at any time.
+Your console will now be displayed in your preferred language, and you will also receive marketing and product communications in this language. You can repeat this procedure to change your preferred language at any time.
```
Comment (jcirinosclwy) [inline console/account/how-to/change-language.mdx:29]:
```suggestion
Your console will now be displayed in your preferred language, which will also be used for all marketing and product communications you receive. You can change your preferred language at any time by repeating this procedure.
```

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**